src/renderer/components/ui/Carousel.tsx

```diff
@@ -0,0 +1,14 @@
+import React from "react";
+
+type Props = {
+  children?: JSX.Element[];
+  page: number;
+};
+
+function Carousel({ children }: Props) {
```

Comment:
Can you more elaborate on this component? is this stub code?

Reply:
Oh, this is a component that was intended to be used in `ActivationWaitSubview`, but the content was not supplied. I'll remove it since it is more like a stub code.

Reply:
Resolved in [053492c](https://github.com/planetarium/9c-launcher/pull/2110/commits/053492c781882a9d4ec624d01190fcc635d5187d)